### PR TITLE
BE: LOK-2651 Bring node info to the event service.

### PR DESCRIPTION
**Pull request by Mohammad-Junid**
…test cases

## Description
Its discussed that  , we have to bring all the node information in event service.

## Jira link(s)
- https://opennms.atlassian.net/browse/LOK-2651

## Checklist
* [ ] Follows Lōkahi's [development guidelines.](https://github.com/OpenNMS-Cloud/lokahi/wiki/Development-Guidelines)
* [ ] Appropriate reviewer(s) have been selected.
* [ ] Jira issue(s) have been updated to "In Review".
* [ ] Includes [appropriate tests.](https://github.com/OpenNMS-Cloud/lokahi/wiki/Test-Strategy)
* [ ] Documentation has been updated as necessary.
* [ ] Notify devops of changes to the Charts.
* [ ] Notify documentation team of any changes to names of screens or features (affects URLs).

### CODE REVIEW

File: events/persistence/src/main/java/org/opennms/horizon/events/persistence/service/EventService.java [hunk 1]
```diff
@@ -74,4 +83,51 @@ public EventLogListResponse searchEvents(String tenantId, EventsSearchBy searchB
...
+
+    public void deleteNodeInEvent(NodeOperationProto nodeOperationProto) {
+
+        var listEventsByNodeId = eventRepository.findAllByTenantIdAndNodeId(
+                nodeOperationProto.getNodeDto().getTenantId(),
+                nodeOperationProto.getNodeDto().getId());
+        listEventsByNodeId.forEach(event -> {
+            event.setNodeId(null);
+        });
+        eventRepository.saveAll(listEventsByNodeId);
```
Comment (cgorantla) [inline events/persistence/src/main/java/org/opennms/horizon/events/persistence/service/EventService.java:126]: probably we can skip this part. Don't need to set node as null. Ideally we can remove events for the node that doesn't exist.

**Review by cgorantla — APPROVED** [review]
LGTM.

**Review by aurang-zeb313 — COMMENTED** [review]
Please  add testing screen shot.

**Comment by Mohammad-Junid** [comment]
**Add Node in Events  Service** 
image
image

**Update Node**
Node Alias also update in events service node
image
image

**Delete Node**
image
image
image

**Review by smunir-onms — COMMENTED** [review]
LGTM

**Review by aurang-zeb313 — APPROVED** [review]
Good work. Approved

**Pull request merged**